docs/upgrade.md

```diff
@@ -5,6 +5,12 @@ description: Guide to update between major Powertools versions
 
 <!-- markdownlint-disable MD043 -->
 
+## End of support v1
+
+On March 31, 2023, AWS Lambda Powertools for Python version 1 will reach end of support. After that, AWS Lambda Powertools for Python version 1 will no longer receive updates or releases. If you are still using v1, we encourage you to read our upgrade guide and update to the latest version.
```

Comment:
```suggestion
On March 31st, AWS Lambda Powertools for Python v1 will reach end of support. After that, Powertools v1 will no longer receive updates or releases. If you are still using v1, we encourage you to read our upgrade guide and update to the latest version.
```

Niicee! Small change so you remain consistent (version 1, v1 ==> v1). You can also use Powertools (shorter) after using the longer version to keep it concise.